WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -1171,6 +1189,57 @@ - (void)presentWebViewControllerWithURL:(NSURL *)URL
 
 #pragma mark - PostHeaderView helpers
 
+- (void)handleFollowingConversationButtonTapped
+{
+    __typeof(self) __weak weakSelf = self;
```

Comment:
I guess this is another piece of logic that we might want to extract to another layer?

Reply:
By extracting to another layer, do you mean just the part that deals with the service? If so, I've swapped over to using the new FollowCommentsService 👍 

If you meant the whole method, I was thinking I _could_  move it out to the ReaderPostHeaderView... what do you think?

Reply:
Now that I'm reading again I can see that most of this logic is related to the presentation.

My main idea was to extract any logic to another layer to avoid the ViewController from calling database/network directly. I think we can keep that way. :)